lib/challenge.js

```diff
@@ -52,6 +91,16 @@ var some_pet;
 
 // Answer Starts Here
 
+var some_pet = {
+  name: 'Claire',
+  species: 'dog',
+  breed: 'Rottweiler',
+  noise: 'Bark',
+  makeNoise: f() {
+    return 'Bark!'
```

Comment:
Mmm.... 
We were looking for you to refer to the `noise` property here; i.e. `return this.noise`.